largest_num.js

```diff
@@ -1,2 +1,25 @@
 alert('hello world');
-prompt('Soke');
\ No newline at end of file
+prompt('Soke');
+
+//Exercise(5) number one(1)
+//Write a function that takes an array of numbers and returns the largest number in the array.
+
+// This is the Pseudocode
+//- we set a function of largestArray to it's prim array.
+//- we set a variable of largestNumber to the first index of the array.
+//- we set a FOR loop by initializing index to 1 and set a condition of the length of am array greater than index and increment index.
+//- we set am IF statement of index of array greater than largestNumber, and equalize both.
+//- we return largestNumber.
+
+// the code.
+function largestArray(array) {
```

Comment:
[Optional]
For a clean code you can name your parameter as numArray instead of array
`function largestArray(numArray) {
    let largestNumber = numArray[0];
    for (let i = 1; i < numArray.length; i++)
      if (numArray[i] > largestNumber)
        largestNumber = numArray[i];
    return largestNumber;
  }`